### DISCUSSION TITLE
New person here... would like to pass a verbosity option to libcamera

### DISCUSSION THREAD

**Discussion by ghost** [bluenviron/mediamtx#2617] [Questions]
### Question

So... simple query.

I noticed that libcamera-vid can pass a verbosity option so that you can get debug information when  running raspberrypi cameras.
How would i go about doing that as part of mediamtx - with the somewhat general requirement of running mediamtx with the attached raspberrypi camera.
I looked in configuration and there's no option for verbosity with rpiCamera configuration.

**Comment by aler9** [comment]
Hello, thanks to #2811 now the `logLevel` option in `mediamtx.yml` is passed to libcamera too. This will be included in the next release, in the meanwhile, you can use this nightly release:

https://github.com/bluenviron/mediamtx/actions/runs/7221295430

[a comment by an automated account is not shown]
